docs/user/metrics/common.py

```diff
@@ -6,20 +6,19 @@
     'TorchGeo',
     'OTB',
     'TerraTorch',
-    'Raster Vision',
     'DeepForest',
+    'Raster Vision',
     'samgeo',
     'spopt',
     'GDL',
     'SITS',
     'SPy',
     'srai',
-    'geo-bench',
```

Comment:
geo-bench has not had a commit in over a year and is now removed from the list. Looking forward to adding geo-bench 2 to the list when it hits 100+ stars and stable releases @nilsleh